string[] pose_names
---
bool plan_success